app/controllers/api/posts_controller.rb

```diff
@@ -0,0 +1,8 @@
+class Api::PostsController < ApplicationController
+  load_and_authorize_resource
+
+  def index
+    @posts = Post.all
+    render json: @posts
+  end
+end
```

Comment:
- [ ] Great job on creating the API for your posts however you need to make sure that when I try to access the URL with a specific user_is like `/api/users/1/posts` it only lists posts from user 1 and not all the posts that exist in the database